litestar/dto/factory/abc.py

```diff
@@ -225,6 +225,9 @@ def _parse_model(
 
         if rename := config.rename_fields.get(field_definition.name):
             field_definition = field_definition.copy_with(serialization_name=rename)  # noqa: PLW2901
+        elif rename_strategy := config.rename_strategy:
+            alias = RenameStrategies(rename_strategy)(field_definition.name)
+            field_definition = field_definition.copy_with(serialization_name=alias)  # noqa: PLW2901
```

Comment:
```suggestion
        elif config.rename_strategy:
            alias = RenameStrategies(config.rename_strategy)(field_definition.name)
            field_definition = field_definition.copy_with(serialization_name=alias)  # noqa: PLW2901
```
I don't think the walrus is necessary here, although if you do want to reference it, it can be done once before the loop.